Index: mysite/catalog/views.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from django.shortcuts import render\n\n\ndef home(request):\n    return render(request, 'catalog/home.html')\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/mysite/catalog/views.py b/mysite/catalog/views.py
--- a/mysite/catalog/views.py	(revision 2945b014b8d817a82f1441cb2c269c482b0da695)
+++ b/mysite/catalog/views.py	(date 1715511728126)
@@ -3,3 +3,4 @@
 
 def home(request):
     return render(request, 'catalog/home.html')
+
Index: mysite/mysite/urls.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\"\"\"\nURL configuration for mysite project.\n\nThe `urlpatterns` list routes URLs to views. For more information please see:\n    https://docs.djangoproject.com/en/5.0/topics/http/urls/\nExamples:\nFunction views\n    1. Add an import:  from my_app import views\n    2. Add a URL to urlpatterns:  path('', views.home, name='home')\nClass-based views\n    1. Add an import:  from other_app.views import Home\n    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')\nIncluding another URLconf\n    1. Import the include() function: from django.urls import include, path\n    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))\n\"\"\"\nfrom django.contrib import admin\nfrom django.urls import include, path\n\nurlpatterns = [\n    path('admin/', admin.site.urls),\n    path('catalog/', include('catalog.urls')),\n]\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/mysite/mysite/urls.py b/mysite/mysite/urls.py
--- a/mysite/mysite/urls.py	(revision 2945b014b8d817a82f1441cb2c269c482b0da695)
+++ b/mysite/mysite/urls.py	(date 1715511728129)
@@ -21,3 +21,4 @@
     path('admin/', admin.site.urls),
     path('catalog/', include('catalog.urls')),
 ]
+
